test/set_schema_test.lua

```diff
@@ -1354,9 +1359,14 @@ g.test_gh_108_fieldno_index_in_space_format = function()
 
     t.assert_equals(box.space['weird_space']:format(),
         {{is_nullable = false, name = "id", type = "unsigned"}})
-    t.assert_equals(
-        box.space['weird_space'].index['pk'].parts,
-        {{fieldno = 1, type = 'unsigned', is_nullable = false}})
+
+    local res = box.space['weird_space'].index['pk'].parts
+    if res[1].exclude_null == nil then
```

Comment:
Let's add a comment here (and below, copypasting here shouldn't be bad for anyone) with the same contents as your commit message so it would be obvious what's going on.

Reply:
Done.